SECURITY.md

```
@@ -0,0 +1,7 @@
+# Reporting Security Issues
+
+To report a security issue or vulnerability, submit a [private vulnerability report via GitHub](https://github.com/HolmesGPT/holmesgpt/security/advisories/new) to the repository maintainers with a description of the issue, the steps you took to create the issue, affected versions, and, if known, mitigations for the issue.
+
+The maintainers will respond within 7 working days of your report. If the issue is confirmed as a vulnerability, we will open a Security Advisory and acknowledge your contributions as part of it. This project follows a 90 day disclosure timeline.
+
+You may also report vulnerabilities or contact us by email: holmesgpt-security@robusta.dev
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)